### DISCUSSION TITLE
Flaky collection of net7 on Windows 10

### DISCUSSION THREAD

**Discussion by szarykott** [microsoft/perfview#1860] [Q&A]
I encountered a strange behavior of PerfView with regard to its `Collect` capability. I am starting a discussion, not an issue since it is quite possible that this is documented somewhere and I just missed it.

Long story short sometimes I see this output when I try to view `CpuStacks`

![obraz](https://user-images.githubusercontent.com/32487860/232297450-e2e22ef7-544c-4348-bf35-90b7a152445f.png)

I use start parameters such as this

![obraz](https://user-images.githubusercontent.com/32487860/232297604-3cdfc5e8-d667-440a-b988-9139fb6722e7.png)

Funny enough, this happens only sometimes, there is never an error of any kind. I did not manage to grasp any pattern:
- I can run 3 traces on a single process - 1st succeeds, 2nd fails, 3rd succeeds (or in any other order)
- running from the command line vs from UI also does not seem to change much

What seems to impact it is the path PerfView itself and the traces it takes reside. After moving from Users/RadosławKot/Documents (notice ł sign that is not ASCII) to C:\some_ascii_folder it works as intended - no matter how hard I try, I am not able to get faulty behavior after the move.

I know it is a bad practice to include non-ascii in paths, but this can happen sometimes and might take some time to debug. In this case, this PC was set up by my company, and unknowingly I started putting traces in the Documents folder.

Is it a known issue?

**Comment by Xhanti** [comment]
Do you by any chance use windows defender with real time protection on? It breaks perfview for me and others, there's in issue in the repo regarding this.  I normally have to turn off real time protection to get stable results, it wasn't always like that but 🤷🏽‍♂️
